phoenix-core/src/it/java/org/apache/phoenix/end2end/LoadSystemTableSnapshotIT.java

```diff
@@ -112,7 +161,16 @@ private static void importSnapshot(String key, String value) throws IOException
     }
 
     @Test
-    public void testDummy() throws Exception {
+    public void testPhoenixUpgrade() throws Exception {
+        Map<String, String> serverProps = Maps.newHashMapWithExpectedSize(2);
+        serverProps.put(QueryServices.EXTRA_JDBC_ARGUMENTS_ATTRIB, QueryServicesOptions.DEFAULT_EXTRA_JDBC_ARGUMENTS);
+        serverProps.put(QueryServices.IS_NAMESPACE_MAPPING_ENABLED, "true");
+        Map<String, String> clientProps = Maps.newHashMapWithExpectedSize(2);
+        clientProps.put(QueryServices.IS_NAMESPACE_MAPPING_ENABLED, "true");
+
+        //Now we can start Phoenix
+        setUpTestDriver(new ReadOnlyProps(serverProps.entrySet().iterator()), new ReadOnlyProps(clientProps.entrySet()
+                .iterator()));
         assertTrue(true);
```

Comment:
Is this assertion enough ?
Will the driver throw an exception if there is an upgrade problem ?

Reply:
I think, yes
If you see the tests results in my [other PR](https://github.com/apache/phoenix/pull/1485), without the PHOENIX-6754 fix, that fails with 
`org.apache.phoenix.schema.ColumnNotFoundException: ERROR 504 (42703): Undefined column. columnName=SYSTEM.CATALOG.COLUMN_QUALIFIER`
Test run on this one is successful.

Will update this and the other PR, based on the things you suggested but it would be better to merge PHOENIX-6754 before

Reply:
That's fine, ping me when the PHOENIX-6754 are finished.

Reply:
PHOENIX-6754 is finished, but we wanted to run these tests on the change
https://github.com/apache/phoenix/pull/1474